cluster/terraform/k8s/modules/argo_workflows/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_s3_bucket" "workflow_artifacts" {
+  bucket = "curi-${var.cluster_name}-workflows"
+}
```

Comment:
it will create a bucket in the test env, I was being lazy about making it only in prod but it won't hurt anything to have an empty bucket in test